app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/ui/view/SecondPageContent.kt

```diff
@@ -0,0 +1,249 @@
+    modifier: Modifier,
+    constraintSet: ConstraintSet,
+    catalogItem: CatalogItem,
+    isFeatureHorizontal: Boolean
+) {
+    ConstraintLayout(
+        constraintSet = constraintSet,
+        modifier = modifier
+    ) {
+        TextDescription(
+            modifier = Modifier.layoutId(SECOND_PAGE_TITLE_TEXT_ID),
+            text = catalogItem.primaryDescription,
+            fontSize = if (isFeatureHorizontal)
+                fontDimensionResource(id = R.dimen.text_size_20) else
+                fontDimensionResource(id = R.dimen.text_size_16),
+            contentDescription = catalogItem.primaryDescription
+        )
+
+        Row(
+            modifier = Modifier
+                .padding(top = dimensionResource(id = R.dimen.catalog_margin_small))
+                .layoutId(SECOND_PAGE_FIRST_ROW_ID),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Image(
+                modifier = Modifier
+                    .padding(start = dimensionResource(id = R.dimen.catalog_image_margin_start))
+                    .clip(MaterialTheme.shapes.small)
+                    .clipToBounds()
+                    .requiredWidth(dimensionResource(id = R.dimen.catalog_min_image_width))
+                    .requiredHeight(dimensionResource(id = R.dimen.catalog_min_image_height)),
+                painter = rememberAsyncImagePainter(model = getImageUri(catalogItem.firstPicture)),
+                contentDescription = catalogItem.firstPictureDescription
+            )
+
+            TextDescription(
+                modifier = Modifier.padding(
+                    top = dimensionResource(id = R.dimen.small_margin),
+                    start = dimensionResource(id = R.dimen.catalog_horizontal_margin),
+                    end = dimensionResource(id = R.dimen.catalog_horizontal_margin)
+                ),
+                text = catalogItem.secondaryDescription ?: "",
+                fontSize = if (isFeatureHorizontal)
+                    fontDimensionResource(id = R.dimen.text_size_16) else
+                    fontDimensionResource(id = R.dimen.text_size_12),
+                contentDescription = catalogItem.secondaryDescription ?: ""
+            )
+        }
+
+        TextDescription(
+            modifier = Modifier
+                .padding(
+                    start = dimensionResource(id = R.dimen.catalog_horizontal_margin),
+                    end = dimensionResource(id = R.dimen.catalog_horizontal_margin),
+                    top = dimensionResource(id = R.dimen.catalog_margin_large)
+                )
+                .layoutId(SECOND_PAGE_THIRD_TEXT_ID),
+            text = catalogItem.thirdDescription ?: "",
+            fontSize = if (isFeatureHorizontal)
+                fontDimensionResource(id = R.dimen.text_size_16) else
+                fontDimensionResource(id = R.dimen.text_size_12),
+            contentDescription = catalogItem.thirdDescription ?: ""
+        )
+
+        Image(
```

Comment:
If we are going to need the image with the same style for the following pages, I would suggest making it a separate component.